libs/sdk-ui-kit/src/RecurrenceForm/RepeatTypeSelect.tsx

```diff
@@ -40,15 +40,28 @@ const getLocalizationKey = (id: RecurrenceType): MessageDescriptor => {
 
 const getRepeatItems = (
     intl: IntlShape,
-    startDate: Date,
+    startDate: Date | null,
     allowHourlyRecurrence?: boolean,
 ): IDropdownItem[] => {
-    const isLastOccurenceOfWeekDay = isLastOccurrenceOfWeekdayInMonth(startDate);
+    const isLastOccurenceOfWeekDay = startDate ? isLastOccurrenceOfWeekdayInMonth(startDate) : false;
```

Comment:
unnecessary condition as isLastOccurrenceOfWeekdayInMonth will return false when date is missing

Reply:
or maybe undefined is causing problems as date may be null?